lib/lhm/throttler/slave_lag.rb

```diff
@@ -44,14 +44,30 @@ def throttle_seconds
       end
 
       def slave_hosts
-        slaves = get_slaves.map { |slave_host| slave_host.partition(':')[0] }
+        @slave_hosts ||= get_slave_hosts
+      end
+
+      def get_slave_hosts
+        slave_hosts = []
```

Comment:
nitpick, the fact that this variable is the same name as one of the methods here might be a little confusing if you missed this declaration line in the first pass.